app/public/map.js

```diff
@@ -361,6 +361,10 @@
     var timesCircleButton = document.createElement('i')
     timesCircleButton.setAttribute('class', 'fa fa-times-circle fa-3x add-form-times')
     timesCircleButton.setAttribute('aria-hidden', 'true')
+    timesCircleButton.addEventListener('click', function (e) {
+      newDiv.classList.toggle('visible')
```

Comment:
you need to also remove the form from the page (same as in the submitForm function), otherwise we will end with multiple forms on the page